daedalus/.babelrc

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
The above binary file looks like it was committed by accident, we should remove it. ^^

Reply:
Please check your comment ^ again, `daedalus/.babelrc` has been removed w/ this PR.

Reply:
I couldn't attach a comment to a binary diff, so that comment referred to that `cabal-config-dist` which is not gone it seems 😉